azurecompute-arm/src/main/java/org/jclouds/azurecomputearm/AzureComputeApi.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecomputearm;
+
+import java.io.Closeable;
+
+import javax.ws.rs.PathParam;
+import org.jclouds.azurecomputearm.features.ResourceGroupApi;
+import org.jclouds.rest.annotations.Delegate;
+
+/**
+ * The Azure Resource Manager API is a REST API for managing your services and deployments.
+ * <p/>
+ *
+ * @see <a href="https://msdn.microsoft.com/en-us/library/azure/dn790568.aspx" >doc</a>
+ */
+public interface AzureComputeApi extends Closeable {
+
+   /**
+    * The Azure Resource Manager API includes operations for managing resource groups in your subscription.
+    *
+    * @see <a href="https://msdn.microsoft.com/en-us/library/azure/dn790546.aspx">docs</a>
+    */
+   @Delegate
+   ResourceGroupApi getResourceGroupApi(@PathParam("subscriptionid") String subscriptionid);
```

Comment:
in `azurecompute` we started from the same idea, but we ended up with https://github.com/jclouds/jclouds-labs/blob/master/azurecompute/src/main/java/org/jclouds/azurecompute/AzureComputeProviderMetadata.java#L71 where the SUBSCRIPTION_ID is part of the endpoint.
Do you think we should follow the same approach or you recommend a pathParam with the subscriptionId, for ARM?